src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final static int BALL_LIMIT = 3;
```

Comment:
let's see the difference with your Lottery class
in Lottery class you have a class field (it is correct), and here you declare this variable inside a method